dojo/api_v2/serializers.py

```diff
@@ -718,10 +719,16 @@ class Meta:
     def create(self, validated_data):
         endpoint = validated_data['endpoint']
         finding = validated_data['finding']
-        status = Endpoint_Status.objects.create(
-            finding=finding,
-            endpoint=endpoint
-        )
+        try:
+            status = Endpoint_Status.objects.create(
```

Comment:
would `get_or_create` work here to make it simpler?

Reply:
Well, It depends on our definition/understanding of the `POST` method. If we will say that
- `POST` request creates a new element if it doesn't exist and updates if exists, we can use `get_or_create` and I can remove `try ... except IntegrityError ...` part.
- `POST` request always creates a new element, we can not use `get_or_create`. Or we can use it but if `create == False` we should raise `ValidationError`.

Reply:
Ah good point, I think we should indeed stay true to the restful meaning of POST